x-pack/plugins/synthetics/public/apps/synthetics/components/settings/private_locations/view_location_monitors.tsx

```diff
@@ -66,7 +66,7 @@ export const ViewLocationMonitors = ({
 const VIEW_LOCATION_MONITORS = i18n.translate(
   'xpack.synthetics.monitorManagement.viewLocationMonitors',
   {
-    defaultMessage: 'View location monitors',
+    defaultMessage: 'View configured monitors',
```

Comment:
```suggestion
    defaultMessage: 'View monitors',
```
Feel free to ignore, I'm just not sure it's needed to use the extra word as the context might provide already enough clarity